src/controllers/wishlistController.ts

```diff
@@ -7,13 +7,13 @@ import User from '../database/models/user';
 
 export const addToWishlist = async (req: Request, res: Response) => {
   try {
-    const { id } = req.params;
-    const user = req.user as User;
+    const { sizeId } = req.params;
+    const { id } = req.user as User;
 
     const itemExist = await Wishlist.findOne({
       where: {
-        userId: user.id,
-        sizeId: id,
+        userId: id,
```

Comment:
size Id can't determine which product being added to a wishList

Reply:
Yeah i changed the parameters i forgot thanks let me fix that